### ISSUE TITLE
Loading 100+ images causes exc_bad_access sometimes

### ISSUE THREAD

**Issue by ozgurshn** [cmtrounce/SwURL#17]
I'm showing 100 + images in a List and sometimes getting this error. What could be the problem? Can you help me please?
Tested on simulator  iPhone 12 (iOS 14.5) and device iPhone 12 Mini (iOS 14.6).

<img width="1139" alt="Screen Shot 2021-07-29 at 11 35 30" src="https://user-images.githubusercontent.com/2404074/127459818-75d16b47-25d9-424e-8aab-db7d040e26f7.png">

![image](https://user-images.githubusercontent.com/2404074/127460567-0e042285-1ddd-484d-9de5-b9f5591d4271.png)

Code is similar to this format
```
List{
                ForEach((0..<500), id: \.self) {
                  index in
                  HStack{
                    RemoteImageView(url: URL(string:"https://picsum.photos/200/300")!)
                    RemoteImageView(url: URL(string:"https://picsum.photos/200/300")!)

                  }

                }
              }
```

**Comment by cmtrounce** [comment]
Hi @ozgurshn , thanks for raising, I'm going to investigate and resolve.

**Comment by cmtrounce** [comment]
Hi @ozgurshn , please point your dependency to the most recent version on the `master` branch.

**Comment by cmtrounce** [comment]
Resolved in release 0.4.1
